openedx/core/djangoapps/catalog/management/commands/sync_course_runs.py

```diff
@@ -0,0 +1,70 @@
+"""
+Sync course runs from catalog service.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.catalog.utils import get_course_runs
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Purpose is to sync course runs data from catalog service to make it accessible in edx-platform.
+    It just happens to only be syncing marketing URLs from catalog course runs for now.
+    """
+    help = 'Refresh marketing urls from catalog service.'
+
+    def update_course_overviews(self, course_runs):
+        """
+        Refresh marketing urls for the given catalog course runs.
+
+        Arguments:
+             course_runs: A list containing catalog course runs.
+        """
+        # metrics for observability
+        # number of catalog course runs retrieved.
+        catalog_course_runs_retrieved = len(course_runs)
+        # number of catalog course runs found in course overview.
+        course_runs_found_in_cache = 0
+        # number of course overview records actually get updated.
+        course_metadata_updated = 0
+
+        for course_run in course_runs:
+            marketing_url = course_run['marketing_url']
```

Comment:
`course_run`  will always have `marketing_url ` key. Its value set to `None` is possible.